ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/ActivitySteps.java

```diff
@@ -193,7 +195,10 @@ public void checkAllDurationOfStep(int indexActivity) {
         int numberOfRow = activityTab.getActivityLogRows(indexActivity - 1).size();
         for (int indexRow = 0; indexRow < numberOfRow; indexRow++) {
             Assertions.assertThat(activityTab.getColumnInRowInActivityLog(indexActivity - 1, indexRow, Activity.COLUMN.DURATION))
-                    .matches("^.*(\\d+ ms|\\d+ seconds)|(-)$");
+                .matches("^.*(\\d+ ms|\\d+ seconds)|(NaN)$"); // NaN is workaround for gh-5715
+            //          prepared reproducer for gh-5715
```

Comment:
do you have a jira issue to track this task?

Reply:
Yes, FUSEQE-4587 and FUSEQE-4588